kansha/batch/send_notifications.py

```diff
@@ -21,7 +21,7 @@ def main(hours, url):
     global apps
     app = apps[APP]
 
-
+    mail_sender = app._services['mail_sender']
```

Comment:
2 problems:
 1. access to a private attribut ('_services')
 2. Demeter's law violation (access to item 'mail_sender' of the foreign object '_services')

=> add a 'send_notifification()' method to the Kansha 'app'